flags/src/main/java/org/secondbase/flags/Flags.java

```diff
@@ -564,11 +564,11 @@ protected SecretPath getS3Path(final Object value) {
             return null;
```

Comment:
I know this isn't part of the change, but, this seems to be an error condition. Throwing an exception would be better than returning null.